spec/lib/ability_spec.rb

```diff
@@ -110,6 +119,7 @@
 
     it_behaves_like "it can manage training requests"
     it_behaves_like "it can read notifications"
+    it_behaves_like "it cannot access problem reservations"
```

Comment:
This one's wrong. The spec wasn't using a `Facility` as its subject resource, so the grants (in the test) weren't right. The next commit I'm pushing up should take care of it (and reflect what we see in the UI for this case).